packages/syft/src/syft/service/action/action_object.py

```diff
@@ -196,7 +196,9 @@ def syft_history_hash(self) -> int:
         return hashes
 
     @classmethod
-    def syft_function_action_from_kwargs_and_id(cls, kwargs, user_code_id):
+    def syft_function_action_from_kwargs_and_id(
+        cls, kwargs: Any, user_code_id: UID
```

Comment:
should be

```suggestion
        cls, kwargs: Dict[str, Any], user_code_id: UID
```

`**kwargs: Any` but `kwargs: Dict[str, Any]`